app/assets/javascripts/discourse/app/widgets/post-stream.js

```diff
@@ -310,7 +286,8 @@ export default createWidget("post-stream", {
             new RenderGlimmer(
               this,
               "div.time-gap.small-action",
-              hbs`<Post::TimeGap @daysSince={{@data.daysSince}} />`,
+              hbs`
+                  <Post::TimeGap @daysSince={{@data.daysSince}} />`,
```

Comment:
```suggestion
              hbs`<Post::TimeGap @daysSince={{@data.daysSince}} />`,
```
undo prettier weirdness